### ISSUE TITLE
Have hover popups for class instantiation also show constructor docstring

### ISSUE THREAD

**Issue by mawright** [microsoft/python-language-server#1537]
## Environment data

- VS Code version: 1.25.1
- Extension version (available under the Extensions sidebar): 2018.6.0
- OS and version: Ubuntu 16.04
- Python version (& distribution if applicable, e.g. Anaconda): Anaconda, 3.6.6
- Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda 
- Relevant/affected Python packages and their versions: N/A

## Actual behavior
Here's some screenshots:

![screenshot from 2018-07-13 14-53-55](https://user-images.githubusercontent.com/2333858/42715618-4fa76a28-86e7-11e8-8e41-6c147ee37937.png)
![screenshot from 2018-07-13 14-54-09](https://user-images.githubusercontent.com/2333858/42715621-51c09b5e-86e7-11e8-84fc-924e7fba939b.png)

Note that in both screenshots, the hover popup only shows the Class docstring.

Note: If there is no class docstring, the hover popup has no docstring info.

## Expected (requested?) behavior

Hover popup would include constructor docstring. This is how e.g., IPython behaves.

```
Python 3.6.6 |Anaconda, Inc.| (default, Jun 28 2018, 17:14:51) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import test

In [2]: test.TestClass?
Init signature: test.TestClass(my_param)
Docstring:      Class docstring
Init docstring: Constructor docstring
File:           ~/Code/test/test.py
Type:           type
```

## Steps to reproduce:
N/A

Thanks for the work on the great extension.

**Comment by jakebailey** [comment]
This looks like an old issue, and has been fixed for a while now:

![image](https://user-images.githubusercontent.com/5341706/64734522-a123ad00-d49b-11e9-98ad-f6ac7ec056a5.png)

Closing.

**Comment by celynwalters** [comment]
Is there some kind of setting which could be overriding this?
It's always been like this for me and I was surprised to find it fixed.

![docstring](https://user-images.githubusercontent.com/3299161/69563755-97451c80-0fa9-11ea-8801-02d657caae44.png)

- VS Code version: 1.40.1
- Extension version (available under the Extensions sidebar): 2019.11.50794
- OS and version: Ubuntu 16.04
- Python version (& distribution if applicable, e.g. Anaconda): Anaconda, 3.7.3
- Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
- Relevant/affected Python packages and their versions: N/A

**Comment by jakebailey** [comment]
Yes, if jedi's enabled, we're not, so it'd be a bug on jedi itself. Thanks for checking.
